### ISSUE TITLE
Use multi-d solution between center points of voxels

### ISSUE THREAD

**Issue by damianjilk2** [damianjilk2/openmc#10]
In a multi-D problem with continuous geometry (aka OpenMC) and a voxelized solution:
Use this capability between center points of voxels

**Comment by gonuke** [comment]
This may be superseded by damianjilk2/openmc#11

**Comment by damianjilk2** [comment]
Keeping issue open until verified (waiting on issue damianjilk2/openmc#9)
